curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-use-code-related-concepts-and-terms/6633032bf786ff8a61b0c482.md

```diff
@@ -0,0 +1,48 @@
+---
+id: 6633032bf786ff8a61b0c482
+title: Task 24
+challengeType: 19
+dashedName: task-24
+---
+
+# --description--
+
+Please answer the question below.
+
+# --question--
+
+## --text--
+
+Which sentence is using `thanks for` correctly?
+
+## --answers--
+
+`Thanks for help me with the project.`
+
+### --feedback--
+
+Remember, `thanks for` should be followed by a gerund or a noun. Here, `help` should be `helping`.
+
+---
+
+`Thanks for to assist in the task.`
+
+### --feedback--
+
+The phrase `thanks for` should not be followed by `to` + verb. It should be a gerund or a noun.
+
+---
+
+`Thanks for assisting with the task.`
+
+---
+
+`Thanks for the assist with the task.`
```

Comment:
This is actually grammatically correct. I wonder if we should use a different example?

Reply:
I remove "the" from the sentence, so the feedback makes more sense.